cms/templates/cms/imagewithlink.html

```diff
@@ -0,0 +1,7 @@
+{% load wagtailimages_tags %}
+
+<a href="{{ self.link }}" target="_blank">
+    
+    {% image self.image original %}
```

Comment:
We're always loading the original image rendition which may not be desirable every time. I do not see any CSS to make this full column width?

Reply:
It is not already in the current design. not sure either required.